pkg/sql/render.go

```diff
@@ -295,14 +293,14 @@ func (p *planner) SelectClause(
 }
 
 // initFrom initializes the table node, given the parsed select expression
-func (r *renderNode) initFrom(
-	ctx context.Context, parsed *tree.SelectClause, scanVisibility scanVisibility,
+func (p *planner) initFrom(
```

Comment:
any particular reason this and other methods moved from the renderNode to the planner?